Test/Float/Channels/SimpleFloatThroughChannel.pml

```diff
@@ -0,0 +1,19 @@
+chan q = [5] of {float};
+
+active proctype sender()
+{
+    /* float literals can be sent explicitly */
+    q!3.14;
+    /* floats can be sent as variable */
+    float x = 2.7;
+    q!x;
+}
+
+active proctype receiver()
+{
+    float a;
```

Comment:
This is a general comment for the new tests, except the "Dumping" one. The "sim" output verifies the prints. However, the model checker output is verified only for errors, but there are no (or no meaningful) checks. It would be great if:

- there was a positive test, which does not yield an error, despite incorporating assertions on the received data.
- there was a negative test, which yields an error on an intentional unsatisfied assert.

```
E.g., :
active proctype bad_receiver(){
float a;
q?a;
printf("first value = %f\n", a);
assert(a > 3.5);
}


```

Reply:
Fixed for some cases